packages/kit/src/exports/vite/index.js

```diff
@@ -566,7 +566,8 @@ function kit({ svelte_config }) {
 						ssrEmitAssets: true,
 						target: ssr ? 'node16.14' : undefined,
 						// don't use the default name to avoid collisions with 'static/manifest.json'
-						manifest: 'vite-manifest.json'
+						manifest: 'vite-manifest.json',
+						minify: 'esbuild'
```

Comment:
IIUC this would always enable minification for server builds, maybe we should respect the user config with
```suggestion
						cssMinify: initial_config.build?.minify == null ? true : !!initial_config.build.minify
```

1. Changed to `cssMinify` so by default, client build will still be minifed, and SSR builds will be unminified. (Default Vite behaviour)
2. Rely on `initial_config.build?.minify` to respect user config, fallback to `true` as default for CSS files.

Reply:
Will this work for users with older versions of Vite or should we require 4.2 as the min?

Reply:
Yeah this needs 4.2, but we can say that if they want this fixed, they can bump the Vite version, so we don't have to enforce it, otherwise it would be a breaking change.